### ISSUE TITLE
Create landing website

### ISSUE THREAD

**Issue by kriptonian1** [keyshade-xyz/keyshade#11]
# Description
Create the landing page of the website using Next.js and typescript. The boilerplate code is already present in `apps/web`. 

Website design's [Figma link](https://www.figma.com/file/4Y2qkiU7yoifHu37iX41n8/keyshade-coming-soon?type=design&node-id=0%3A1&mode=dev)

> **Note:** this website must have a lighthouse score of above 90

## metadata structure
```tsx
import { Metadata } from "next";

const description = "Manage all your secrets securely with public key encryption and realtime based tools, that seamlessly fits into your codebase";
const name = "keyshade";

export const metadata: Metadata = {
  metadataBase: new URL("https://keyshade.xyz/"),
  title: {
    default: name,
    template: `%s | ${name}`,
  },
  description: description,
  keywords: [
"Keyshade",
"secrets integration",
"codebase security",
"Public Key Encryption",
"secret rotation",
"auto revoke",
"collaboration",
"data security",
"secrets management",
"secure code secrets",
"realtime secrets integration",
"encryption for secrets",
"automated secret rotation",
"team collaboration secrets",
"secure runtime environment",
"secrets management tool",
"codebase security solution",
"cybersecurity for secrets",
"API security",
"key management",
"secure data transfer",
"secure secrets sharing",
"effortless secrets management",
"Keyshade features",
"cybersecurity solution",
"secure software development",
"automated access revocation",
"secure secrets storage",
"developer tools security",
"API key security",
"codebase encryption",
"team-based secrets management"
  ],
  applicationName: name,
  openGraph: {
    title: name,
    description: description,
    type: "website",
    locale: "en_US",
    images: "https://keyshade.xyz/thumbnail.png",
  },
  twitter: {
    card: "summary_large_image",
    title: name,
    description: description,
    creator: "@keyshade_xyz",
    creatorId: "1738929014016966656",
  },
  robots: {
    index: true,
    follow: true,
    noarchive: true,
    nocache: true,
    googleBot: {
      index: true,
      follow: true,
    },
  },
};
```

**Comment by SRINJAYDASGUPTA-Git** [comment]
could i start with this?

**Comment by rajdip-b** [comment]
Heyo ser! @SRINJAYDASGUPTA-Git go ahead!

**Comment by rajdip-b** [comment]
@kriptonian1 brother would you please mail our friend here the required assets?

**Comment by kriptonian1** [comment]
@SRINJAYDASGUPTA-Git can download the assets from Figma itself

**Comment by SRINJAYDASGUPTA-Git** [comment]
> @SRINJAYDASGUPTA-Git can download the assets from Figma itself

okk thanks

**Comment by kriptonian1** [comment]
Btw @SRINJAYDASGUPTA-Git do use SVGs, and to handle SVGs do use `svgr`, also don't add background as an image

**Comment by rajdip-b** [comment]
Okei friend let us know if you get stuck somewhere! Feel free to open up a ticket on our discord channel to do a conversation on this! We are always available for feedbacks!

**Comment by SRINJAYDASGUPTA-Git** [comment]
> Btw @SRINJAYDASGUPTA-Git do use SVGs, and to handle SVGs do use `svgr`, also don't add background as an image

ok 👍👍

**Comment by kriptonian1** [comment]
@SRINJAYDASGUPTA-Git also use tailwind no css

**Comment by SRINJAYDASGUPTA-Git** [comment]
> @SRINJAYDASGUPTA-Git also use tailwind no css

obviously 🙃

**Comment by kriptonian1** [comment]
and for font `next/font`

**Comment by SRINJAYDASGUPTA-Git** [comment]
> and for font `next/font`

if i run pnpm install these will be installed right?
@kriptonian1

**Comment by SRINJAYDASGUPTA-Git** [comment]
@kriptonian1 how to solve this
![image](https://github.com/keyshade-xyz/keyshade/assets/70833470/bc0bbe8a-8a15-4149-b019-a2ef5fb32b7c)

**Comment by rajdip-b** [comment]
Real sorry about that! You would need to install `nx` to run it. You can do it by running `pnpm i -g nx`. Imma add that to the docs real quick.

**Comment by SRINJAYDASGUPTA-Git** [comment]
> Real sorry about that! You would need to install `nx` to run it. You can do it by running `pnpm i -g nx`. Imma add that to the docs real quick.

Ok ok thanks

**Comment by SRINJAYDASGUPTA-Git** [comment]
![image](https://github.com/keyshade-xyz/keyshade/assets/70833470/1e5cc57a-c222-4bb4-b0a7-22f7cc8c112f)
🥲
Help @kriptonian1 @rajdip-b

**Comment by rajdip-b** [comment]
Yes! exactly this issue. @Abbhiishek would you be able to help us?

**Comment by rajdip-b** [comment]
Hey! @SRINJAYDASGUPTA-Git can you do the following and let me know if it works for you?
- delete the `pnpm-lock.json` and `node_modules`
- run `pnpm i`
- run `pnpm run dev:api`

**Comment by SRINJAYDASGUPTA-Git** [comment]
> Hey! @SRINJAYDASGUPTA-Git can you do the following and let me know if it works for you?
> - delete the `pnpm-lock.json` and `node_modules`
> - run `pnpm i`
> - run `pnpm run dev:api`

On it

**Comment by rajdip-b** [comment]
You can also go through this article: https://github.com/nrwl/nx/issues/10172. We need to document this issue if it persists :/

**Comment by Abbhiishek** [comment]
> Yes! exactly this issue. @Abbhiishek would you be able to help us?

Hey i thinks its a `pnpm install` issue. as you suggested removing node modules might help. I have not encountered such issue yet.

**Comment by rajdip-b** [comment]
For future readers, the issue was caused due to the `pnpm-lock.yaml` file. I have excluded it via `.gitignore`

**Comment by rajdip-b** [comment]
![image](https://github.com/keyshade-xyz/keyshade/assets/83924254/e852542b-ec04-4725-b686-3bbeb8a0d8b8)
Please update the metadata here as mentioned in #11

![image](https://github.com/keyshade-xyz/keyshade/assets/83924254/9ccfe2eb-b186-4a91-88b0-443ff9499f53)
Also, update the logo!

**Comment by rajdip-b** [comment]
@kriptonian1 can you please let us know about what the inconsistencies are? With the figma design?

**Comment by rajdip-b** [comment]
Fixed by #34
